Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -550,6 +551,49 @@
     m_frame->protectedPage()->send(Messages::WebPageProxy::SetCursor(WebCore::Cursor::fromType(cursorType)));
 }
 
+bool PDFPluginBase::handleContextMenuEvent(const WebMouseEvent& event)
+{
+    if (!m_frame || !m_frame->coreLocalFrame())
+        return false;
+    RefPtr webPage = m_frame->page();
+    if (!webPage)
+        return false;
+    RefPtr frameView = m_frame->coreLocalFrame()->view();
+    if (!frameView)
+        return false;
+
+    IntPoint point = frameView->contentsToScreen(IntRect(frameView->windowToContents(event.position()), IntSize())).location();
+
+    NSMenu *nsMenu = createContextMenu(event, webPage->userInterfaceLayoutDirection());
+
+    if (!nsMenu)
+        return false;
+
+    std::optional<int> openInPreviewIndex;
+    Vector<PDFContextMenuItem> items;
+    auto itemCount = [nsMenu numberOfItems];
+    for (int i = 0; i < itemCount; i++) {
+        auto item = [nsMenu itemAtIndex:i];
+        if ([item submenu])
+            continue;
+
+        // FIXME: This second part of this condition should be removed once UnifiedPDFPlugin::openWithPreview is implemented
+        if ([NSStringFromSelector(item.action) isEqualToString:@"openWithPreview"] && !isUnifiedPDFPlugin())
```

Comment:
Use `sel_isEqual()` rather than string comparison. But it's a bit weird to compare selectors. Usually NSMenuItems etc have tags and you compare by tag.